indexing-service/src/main/java/io/druid/indexing/common/TaskLock.java

```diff
@@ -31,19 +31,25 @@
   private final String dataSource;
   private final Interval interval;
   private final String version;
+  private final Integer priority;
+  private final Boolean preemptive;
 
   @JsonCreator
   public TaskLock(
       @JsonProperty("groupId") String groupId,
       @JsonProperty("dataSource") String dataSource,
       @JsonProperty("interval") Interval interval,
-      @JsonProperty("version") String version
+      @JsonProperty("version") String version,
+      @JsonProperty("priority") Integer priority,
+      @JsonProperty("preemptive") Boolean preemptive
```

Comment:
"upgraded" would also match the names of the upgradeLock methods, and the name of the LockUpgradeAction.